homeassistant/components/elkm1/climate.py

```diff
@@ -53,24 +58,25 @@ class ElkThermostat(ElkEntity, ClimateEntity):
         | ClimateEntityFeature.AUX_HEAT
         | ClimateEntityFeature.TARGET_TEMPERATURE_RANGE
     )
+    _element: Thermostat
 
-    def __init__(self, element, elk, elk_data):
+    def __init__(self, element: Element, elk: Elk, elk_data: dict[str, Any]) -> None:
         """Initialize climate entity."""
         super().__init__(element, elk, elk_data)
-        self._state = None
+        self._state: str = HVAC_MODE_OFF
```

Comment:
If we don't know the state we should still set it to None

str | None

If elk always reports it as off if it doesn't know then this is ok

Reply:
The lib defaults the thermostat to off before syncing with the panel. I'm OK picking off here too.

Reply:
Sounds good